### PR TITLE
Update TAU Image

### PR DESCRIPTION
**Pull request by singa2000**
Some api images are missing so it add.

Signed-off-by: cookie <cookie@samsung.com>

### Change Description ###

Describe your changes here.

For instance,
 - Fix broken links of native application docs.
 -  ...

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_PanelChanger.htm [hunk 1]
```diff
@@ -14,6 +14,7 @@ <h1>Panel Changer</h1>
 <p>Panel can have header, content and so on like the page component.<br>
 It works even if the panel components is in other HTML files.<br>
 Panel changer component controls lifecycle of the panels.</p>
+<img src="../../images/mobile_panelchanger.png" alt="photo of panelchanger element in Tizen">
 
 <button
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_PanelChanger.htm]: Maybe a line break is needed here.

The "Use example" button is right next to the image.

image

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm [hunk 2]
```diff
@@ -14,6 +14,7 @@ <h1>Circular Index Scroll Bar</h1>
 <p>If you spin the rotary quickly, an indicator will be displayed and you can move in an index unit.</p>
 <p>The indicator will disappear if no rotary action is detected for a second.</p>
 <p>This component fires a select event when the index characters are selected.</p>
+<img src="../../images/Circularindexscrollbar.png" alt="photo of circleindex element in Tizen">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm]: Maybe a line break is needed here.

The "Use example" button is right next to the image.

image

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm [hunk 3]
```diff
@@ -13,9 +13,7 @@ <h1>Toggle Switch</h1>
 
 <p>The toggle switch component shows on the screen a 2-state switch.</p>
 
-<video autoplay muted loop controls>
-	<source src="../../images/toggleswitch.mp4">
-</video>
+<img src="../../images/Toggleswitch.png" alt="photo of Toggleswitch element in Tizen">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm]: Are you going to remove the mp4 file and use the png file instead?
If so, how about removing not used toggleswitch.mp4?

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm [hunk 4]
```diff
@@ -13,9 +13,7 @@ <h1>Toggle Switch</h1>
 
 <p>The toggle switch component shows on the screen a 2-state switch.</p>
 
-<video autoplay muted loop controls>
-	<source src="../../images/toggleswitch.mp4">
-</video>
+<img src="../../images/Toggleswitch.png" alt="photo of Toggleswitch element in Tizen">
```
Comment (mijin85cho) [inline docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm]: Are you going to remove the mp4 file and use the png file instead?
If so, how about removing not used toggleswitch.mp4?

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_PanelChanger.htm (hunk 1 above)
Comment (singa2000) [reply docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_PanelChanger.htm]: Done

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm (hunk 2 above)
Comment (singa2000) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm]: Done

File: docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm (hunk 4 above)
Comment (singa2000) [reply docs/application/web/api/5.5/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm]: Done

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm (hunk 3 above)
Comment (singa2000) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm]: Done

**Comment by singa2000** [comment]
We have completed the update for the commnet.

**Pull request merged**